entry_types/scrolled/app/views/pageflow_scrolled/entries/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<%= content_tag(:html, lang: I18n.locale) do %>
```

Comment:
**suggestion:** We can also port the [corresponding test from paged](https://github.com/codevise/pageflow/blob/f51a288a5abe2df495ee644e80a64ff8b345a0b6/entry_types/paged/spec/controllers/pageflow_paged/entries_controller_spec.rb#L169).